xml/System.Diagnostics.Contracts/Contract.xml

```diff
@@ -1220,7 +1220,7 @@
    
   
 ## Examples  
- The following example shows how to use the <xref:System.Diagnostics.Contracts.Contract.Result``1*> method to specify an expected return value. This code example is part of a larger example provided for the <xref:System.Diagnostics.Contracts.ContractClassAttribute> class.  
+ The following example shows how to use the <xref:System.Diagnostics.Contracts.Contract.Result``1%2A> method to specify an expected return value. This code example is part of a larger example provided for the <xref:System.Diagnostics.Contracts.ContractClassAttribute> class.  
```

Comment:
This link didn't work. Perhaps it should be System.Diagnostics.Contracts.Contract.Result%2A?

This is the ms.assetid that shows up on the page source: 
![image](https://user-images.githubusercontent.com/12971179/32456422-6129f26a-c2da-11e7-9ffb-86a82e4ffb28.png)